build.gradle

```diff
@@ -358,6 +358,7 @@ project(':iceberg-core') {
     implementation libs.jackson.databind
     implementation libs.caffeine
     implementation libs.roaringbitmap
+    implementation libs.redission
```

Comment:
I don't think that we should move forward with a PR that adds Redis to the classpath. This changes the classpath that will be included in the runtime bundles, so at a minimum, we need to either make it optional or update all of the LICENSE and NOTICE files to note the inclusion.

Reply:
@rdblue So, we only keep ServerSideHttpLockManager and delete RedissonLockManager, is it OK?

Reply:
@rdblue Sir, I deleted all the Redisson-related parts.